communications/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+- Give all participants as much context as possible before the meeting.
+- _Asynchronously_ resolve as many items as possible before the meeting.
+- Determine, and possibly _reduce_, the scope, and length of the meeting.
+- Determine, and possibly _eliminate_, reasons to participate in the meeting.
+- Prepare the meeting set up to be as convenient as possible for all participants.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
```

Comment:
Does this one mean everyone should participate in taking meeting notes?  Don't fully understand this sentence.

Reply:
I left things open to avoid unnecessary contest on being too rigid on the process, and/or may-or-may-not applicable legal implications. The execution very much depends on the willpower of the actual participants.

Essentially, this should be a gateway toward less-to-no meetings. The pre-meeting gets the most attention conveys this point -- most meetings can be avoided. But if it doesn't achieve this purpose, at least it betters the conventional meeting set up drastically.